test/licensemaster/obsoletelm_m4_test.go

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) 2018-2022 Splunk Inc. All rights reserved.
```

Comment:
Not sure if we need to change the name here to obsolete

Reply:
Resolved in the latest commit.